go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/golang/protobuf v1.3.2
 	github.com/google/go-cmp v0.3.1
 	github.com/jhump/protoreflect v1.5.1-0.20191024142022-1d0024204e58
+	github.com/kjk/inflect v0.0.0-20190213232030-bfc9dcd6393b
```

Comment:
Can you use `github.com/gertd/go-pluralize` instead?

Reply:
+1, please use the dependency we already have.

Reply:
Done.